Formula/rex.rb

```diff
@@ -205,10 +227,18 @@ class Rex < Formula
     sha256 "771f7d02abd1ded94d9e37d3f66e795c8d2026d04defbeb5b679ca058116bbf3"
   end
 
+  resource "inc::latest" do
+    url "http://search.cpan.org/CPAN/authors/id/D/DA/DAGOLDEN/inc-latest-0.500.tar.gz"
+    sha256 "daa905f363c6a748deb7c408473870563fcac79b9e3e95b26e130a4a8dc3c611"
+  end
+
   def install
     ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
     ENV.prepend_path "PERL5LIB", libexec/"lib"
 
+    # Spare Perl v5.23+ users a long-winded gripe from JSON::XS module's author.
```

Comment:
The only issue is a petty, preachy warning during a build process. You honestly expect them to revise their project's dependencies over this? Hell, they'd probably ignore it because `JSON::XS` has a lot going for it in terms of speed and performance. Not worth giving up over a non-issue.